src/main/java/com/example/bookstoreproject/configuration/JwtTokenAuthorizationFilter.java

```diff
@@ -0,0 +1,47 @@
+package com.example.bookstoreproject.configuration;
+
+import com.example.bookstoreproject.domain.auths.JwtTokenService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.HttpHeaders;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import static org.apache.commons.lang3.StringUtils.startsWith;
+
+
+@Configuration
+@RequiredArgsConstructor
+@Profile("!TEST")
+public class JwtTokenAuthorizationFilter extends OncePerRequestFilter {
```

Comment:
```suggestion
public class JwtRequestFilter extends OncePerRequestFilter {
```

JWT = JSON Web Token, không cần suffix nó với Token nữa